cli/slashingExport_test.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2022 Nethermind
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cli_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/NethermindEth/sedge/cli"
+	"github.com/NethermindEth/sedge/cli/actions"
+	"github.com/NethermindEth/sedge/cli/actions/mock"
+	"github.com/NethermindEth/sedge/configs"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSlashingExport_ValidatorIsRequired(t *testing.T) {
+	tests := []struct {
+		name        string
+		args        []string
+		expectedErr error
+	}{
+		{
+			name:        "without flags",
+			args:        []string{},
+			expectedErr: cli.ErrInvalidNumberOfArguments,
+		},
+		{
+			name:        "with flags",
+			args:        []string{"--network", "sepolia", "--out", "slashing_data.json"},
+			expectedErr: cli.ErrInvalidNumberOfArguments,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {})
+
+		slashingImportCmd := cli.SlashingExportCmd(nil)
+		slashingImportCmd.SetArgs(tt.args)
+		err := slashingImportCmd.Execute()
+		assert.ErrorIs(t, err, tt.expectedErr)
```

Comment:
Copy-paste detected. Wrong variables names and wrong `Execute` call

Reply:
The `slaashingImportCmd` is initialized with the `SlashingExportCmd`, so it's an export command regardless of the name and that means the `Execute` is running the export command as the test requires. In this case, the solution would be to rename the variable to `slashingExportCmd`. Fixed here https://github.com/NethermindEth/sedge/pull/209/commits/94ad27c973701965d682e605df3e6dc263812b7a